data/json/recipes/armor/arms.json

```diff
@@ -24,7 +24,7 @@
     "time": "90 m",
     "autolearn": true,
     "byproducts": [ [ "scrap_cotton", 79 ] ],
-    "using": [ [ "tailoring_cotton_patchwork", 1 ] ]
+    "using": [ [ "tailoring_cotton_patchwork", 1 ], [ "sewing_standard", 20 ] ]
```

Comment:
Tailoring requirements already include thread and other necessary tools, so sewing_standard is not necessary here. Amount of thread demanded by requirement *may* be off, however, and depending on recipe, extra tools may be necessary.

Reply:
fixed